R/what.R

```diff
@@ -18,6 +18,13 @@
 #' what_is_it(1:10)
 #' what_is_it(mtcars)
 what_is_it <- function(x) {
```

Comment:
deprecate_stop ??

Reply:
The more I thought about it, the more I felt that there was nothing to lose by including one cycle of a warn deprecation 🤷🏼  We can hard deprecate after the release?

Reply:
Okay sounds good!   I thought I was being the bad guy and deviating from our process, but here is Daniel following admiral byzantine processes.  Have we won?

Reply:
HA! It's possible!